modules/fiftyone_degrees/device_detection/hook_auction_entrypoint.go

```diff
@@ -0,0 +1,27 @@
+package device_detection
+
+import (
+	"github.com/prebid/prebid-server/v2/hooks/hookexecution"
+	"github.com/prebid/prebid-server/v2/hooks/hookstage"
+)
+
+// handleAuctionEntryPointRequestHook is a hookstage.HookFunc that is used to handle the auction entrypoint request hook.
+func handleAuctionEntryPointRequestHook(cfg Config, payload hookstage.EntrypointPayload, deviceDetector deviceDetector, evidenceExtractor evidenceExtractor, accountValidator accountValidator) (result hookstage.HookResult[hookstage.EntrypointPayload], err error) {
```

Comment:
We need to extract the evidence in the entrypoint hook because it has access to the request headers.
The raw auction hook only has access to the request body.